interview-questions.md

```diff
@@ -8,33 +8,45 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What does calling super do in a Ruby class?
 
-Your answer:
+Your answer: It is passed along with the invoked method.
 
-Researched answer:
+Researched answer: The INITIALIZE method INVOKE with an argument.
```

Comment:
I would review what super is doing for you, as this explanation does not fully answer the question